src/backend/TrafficCourts/Citizen.Service/Models/Dispute/ViolationTicket.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Text.Json.Serialization;
+using TrafficCourts.Common.Converters;
+
+namespace TrafficCourts.Citizen.Service.Models.Dispute
+{
+    public class ViolationTicket
+    {
+        public string? TicketNumber { get; set; }
+        public string? Surname { get; set; }
+        public string? GivenNames { get; set; }
+        public string? DriversLicenceNumber { get; set; }
+        public string? DriversLicenceProvince { get; set; }
+        [JsonConverter(typeof(DateOnlyJsonConverter))]
+        [SwaggerSchema(Format = "date")]
+        public DateOnly? Birthdate { get; set; }
+        public string? Address { get; set; }
+        public string? City { get; set; }
+        public string? Province { get; set; }
+        public string? PostalCode { get; set; }
+        public DateTime? IssuedDate { get; set; }
+        public string? EnforcementOrganization { get; set; }
+        public IList<TicketCount> ViolationTicketCounts { get; set; } = new List<TicketCount>();
+    }
+
+    public class TicketCount
```

Comment:
We already have [TrafficCourts.Citizen.Service.Models.Tickets.ViolationTicketCount](TrafficCourts.Citizen.Service.Models.Tickets.ViolationTicketCount)

Reply:
Changed it to use the existing model TrafficCourts.Citizen.Service.Models.Tickets.ViolationTicketCount